src/main/features/certificateOfService/views/includes/file-upload-form.njk

```diff
@@ -0,0 +1,10 @@
+<form action="/legal/certificateOfService/file-upload?_csrf={{ csrf }}" novalidate method="post" enctype="multipart/form-data">
```

Comment:
why is the csrf token in the query string?

Reply:
when doing a multipart/form-data form it doesn't work if you have the csrf as a field in the form. after googling I found the above solution